bin/mycroft-help

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SOURCE="${BASH_SOURCE[0]}"
+cd -P "$( dirname "$SOURCE" )"/..
+DIR="$( pwd )"
+
+echo -e "\e[36mMycroft\e[0m is your open source voice assistant.  Full source"
+echo "can be found at: ${DIR}"
+echo
+echo "Mycroft-specific commands you can use from the Linux command prompt:"
+echo "  mycroft-cli-client       command line client, useful for debugging"
+echo "  mycroft-msm              Mycroft Skills Manager, to manage your Skills"
+echo "  mycroft-msk              Mycroft Skills Kit, create and share Skills"
```

Comment:
It seems like these two wrappers don't exist yet

Reply:
Added